libraries/chain/include/steem/chain/smt_objects/smt_token_object.hpp

```diff
@@ -110,6 +110,21 @@ class smt_token_object : public object< smt_token_object_type, smt_token_object
 
    ///parameters for 'smt_setup_operation'
    int64_t                       max_supply = 0;
+};
+
+class smt_ico_object : public object< smt_ico_object_type, smt_ico_object >
```

Comment:
What is the difference between this object and the `smt_event_token_object`? They look like they contain, by and large, the same fields. Trying to keep that up to date in multiple places seems like a bad idea.